src/app/shared/faq-assistant/faq-assistant.component.html

```diff
@@ -5,21 +5,32 @@
         <div>Online help</div>
       </div>
       <div class="component-style__content">
-        <div class="component-style__inner" *ngIf="!showFinish">
-          <div class="component-style__inner-title">Ask please what do you want?</div>
-          <div class="component-style__items">
-            <ul>
-              <li class="component-style__item" *ngFor="let item of titleList">
-                <span [id]="item.id" (click)="setTitle(item.item, item.answer)">{{item.title}}</span>
-              </li>
-            </ul>
+        <div>
+          <div class="component-style__inner" *ngIf="!showFinish">
```

Comment:
You use very common names - `component-style`. Please change to a more specific name that will describe the entity.

Reply:
was applied